### ISSUE TITLE
[Bug] cursor keys don't work in theme selection

### ISSUE THREAD

**Issue by Disonantemus** [Beastwick18/nyaa#8]
**Help: Theme** _PopUp_ says:
~~~yaml
┌Help: Theme───────────────────────┐
│      Key             Action      │
│          Enter ⇒ Confirm         │
│      Esc, t, q ⇒ Close           │
│           j, ↓ ⇒ Down            │
│           k, ↑ ⇒ Up              │
│              g ⇒ Top             │
│              G ⇒ Bottom          │
└──────────────────────────────────┘
~~~
But to go up and down to choose a theme, only works: `j`, `k` (and not `↑` and `↓` cursor keys).

**Comment by Beastwick18** [comment]
Very weird. When I make popups, they're mostly copy-paste from the others since their implementation is really simple. I think there must've been a point when popup didn't support arrow keys, and I forgot to add it to this one. I'll fix this.

**Comment by Beastwick18** [comment]
Fixed with f5ac781
